bodhi/tests/functional/test_updates.py

```diff
@@ -1950,6 +1950,48 @@ def test_invalid_stable_request(self, *args):
 
```

Comment:
@lmacken I was thinking do we really need two separate test cases to test the following? :
* Test request to stable before an update reaches stable karma and after it reaches stable karma when autokarma is disabled
* Test manually push to stable when autokarma is disabled and karma threshold is reached

I'm not sure merging these two testcases into one is a good idea. Or should we keep them as two separate test cases them like they are now.